ConfigSpace/configuration_space.pyx

```diff
@@ -1341,6 +1344,30 @@ class ConfigurationSpace(collections.abc.Mapping):
         """
         self.random = np.random.RandomState(seed)
 
+    def remove_parameter_priors(self) -> 'ConfigurationSpace':
+        """
+        Produces a new ConfigurationSpace where all priors on parameters are removed.
+        Non-uniform pararmeters are replaced with uniform ones, and 
+        CategoricalHyperparameters with weights have their weights removed. 
+        
+        Returns
+        -------
+        :class:`~ConfigSpace.configuration_space.ConfigurationSpace`
+            The resulting configuration space, without priors on the hyperparameters
+        """
+        uniform_types = (UniformFloatHyperparameter, UniformIntegerHyperparameter, OrdinalHyperparameter)
+        uniform_config_space = ConfigurationSpace()
+        for parameter in self.get_hyperparameters():
+            if isinstance(parameter, uniform_types):
```

Comment:
Could we check here whether the function `to_uniform` exists instead? Or add `to_uniform` to all hyperparameters, and they would just return a copy of themselves if they are uniform?

With the current code I would be worried that we forget to add new uniform types.